docs/_source/components-overview/pop_syn/binary_star.rst

```diff
@@ -22,98 +25,191 @@ BINARYPROPERTIES
+    -  The mass transfer history for the HMS_HMS phase.
+  * - ``mt_history_CO_HMS_RLO``
+    - The mass transfer history for the CO_HMS_RLO phase.
+  * - ``mt_history_CO_HeMS``
+    - The mass transfer history for the CO_HeMS phase.
+  * - ``mt_history_CO_HeMS_RLO``
+    - The mass transfer history for the CO_HeMS_RLO phase.
 
 
 State options
 ~~~~~~~~~~~~~
 
 Binary states are defined according to the following table:
 
-.. csv-table:: States
-   :header: "State", "Description"
-   :widths: 10, 30
+.. list-table:: States
+  :header-rows: 1
+  :widths: 10 30
+
+  * - State
+    - Description
+  * - ``initially_single_star``
+    - The binary was initially a single star.
+  * - ``detached``
+    - The stars in the binary are in a detached state.
+  * - ``RLO1``
+    - The binary is Roche Lobe overflowing, star 1 is overfilling the RL.
+  * - ``RLO2``
+    - The binary is Roche Lobe overflowing, star 2 is overfilling the RL.
+  * - ``contact``
+    - The stars in the binary are in contact.
+  * - ``disrupted``
+    - The binary was disrupted.
+  * - ``merged``
+    - The stars in the binary merged.
+  * - ``initial_RLOF``
+    - The binary is in the initial Roche Lobe overflow.
+  * - ``maxtime``
+    - Max time of the evolution was reached.
+  * - ``FAILED``
+    - The evolution failed.
 
-   `detached`, "The stars in the binary are in a detached state."
-   `contact`, "The stars in the binary are in contact."
-   `RLO1`, "The binary is Roche Lobe overflowing, star 1 is overfilling the RL."
-   `RLO2`, "The binary is Roche Lobe overflowing, star 2 is overfilling the RL."
-   `CE`, "The binary is in a Common Envelope phase."
-   `disrupted`, "The binary was disrupted."
-   `MaxTimeChanged`, "Max time of the evolution was reached."
-
-TODO: update properties
 
 Event options
 ~~~~~~~~~~~~~
 
 Binary events are defined according to the following table:
 
-.. csv-table:: Events
-  :header: "State", "Description"
-  :widths: 10, 30
+.. list-table:: Events
+  :header-rows: 1
+  :widths: 10 30
+
+  * - Event
+    - Description
+  * - ``ZAMS``
+    - Zero Age Main Sequence
+  * - ``CC1``
+    - Core collapse of star 1.
+  * - ``CC2``
+    - Core collapse of star 2.
+  * - ``oRLO1``
+    - The binary is at onset of Roche Lobe overflow, star 1 is overfilling the RL.
+  * - ``oRLO2``
+    - The binary is at onset of Roche Lobe overflow, star 2 is overfilling the RL.
+  * - ``oCE1``
+    - The binary is at the onset of Common Envelope initiated by star 1.
+  * - ``oCE2``
+    - The binary is at the onset of Common Envelope initiated by star 2.
+  * - ``oDoubleCE1``
+    - The binary is at the onset of Double Common Envelope initiated by star 1. 
+      Both stars are post main-sequence.
+  * - ``oDoubleCE2``
+    - The binary is at the onset of Double Common Envelope initiated by star 2. 
+      Both stars are post main-sequence.
+  * - ``CO_contact``
+    - The binary reached contact in the compact object phase.
+  * - ``redirect_from_ZAMS``
```

Comment:
Just making a note that we should probably add more info about the redirect steps here once we merge PR #421, or we should add changes to the docs in that PR if we accept this first.

Reply:
I would suggest accepting this PR first and making the changes in PR421 or in a separate PR 👍